test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -96,6 +104,170 @@ public void CanChangeEngineFromScriptEditorDropDown()
+            var revertButton = scriptWindow.FindName("RevertScriptChangesButton") as Button;
+
+            //Press the run button
+            runButton.RaiseEvent(new RoutedEventArgs(ButtonBase.ClickEvent));
+            DispatcherUtil.DoEvents();
+
+            //Check that the changes are reflected in the Watch
+            Assert.That(watchOUT.CachedValue,Is.EqualTo("Softdesk,1997"));
+
+            //Pressing the Revert button
+            revertButton.RaiseEvent(new RoutedEventArgs(ButtonBase.ClickEvent));
+            DispatcherUtil.DoEvents();
+            ViewModel.HomeSpace.Run();
+            
+            //After pressing the Revert button we need to have the previous result in Watch node
+            Assert.That(watchOUT.CachedValue, Is.EqualTo("Autodesk,1982"));
+
+            codeEditor.Text = codeEditor.Text.Replace("Softdesk","Autodesk").Replace("1997", "1982");
+            saveButton.RaiseEvent(new RoutedEventArgs(ButtonBase.ClickEvent));
+            ViewModel.HomeSpace.Run();
+
+            Assert.That(watchOUT.CachedValue, Is.EqualTo("Autodesk,1982"));
+        }
+
+        /// <summary>
+        /// This test method will validate that the OnMoreInfoEvent was executed correctly
+        /// </summary>
+        [Test]
+        public void OnMoreInfoEventTest()
+        {
+            Open(@"core\python\python_check_output.dyn");
+            ViewModel.HomeSpace.Run();
+
+            var nodeView = NodeViewWithGuid("3bcad14e-d086-4278-9e08-ed2759ef92f3");
+            var nodeModel = nodeView.ViewModel.NodeModel as PythonNodeBase;
+            Assert.NotNull(nodeModel);
+
+            //Get the Watch node so we can check the content later
+            var watchOUT = Model.CurrentWorkspace.NodeFromWorkspace<Watch>("714838b43a114f5f93cfe2f1d26092cf");
+
+            var scriptWindow = EditPythonCode(nodeView, View);
+            var codeEditor = FindCodeEditor(scriptWindow);
+            codeEditor.Text = codeEditor.Text.Replace("Autodesk", "Softdesk").Replace("1982", "1997");
+            codeEditor.Focus();
+
+            //Check that we don't have any extension tabs shown right now
+            Assert.That(this.View.ExtensionTabItems.Count, Is.EqualTo(0));
+
+            var moreInfoButton = scriptWindow.FindName("MoreInfoButton") as Button;
+
+            //Pressing the MoreInfo button, here the OnMoreInfoEvent is raised 
+            moreInfoButton.RaiseEvent(new RoutedEventArgs(ButtonBase.ClickEvent));
+            DispatcherUtil.DoEvents();
+
+            //Check that now we are showing a extension tab (DocumentationBrowser)
+            Assert.That(this.View.ExtensionTabItems.Count, Is.EqualTo(1));
+
+            var docBrowser = this.View.ExtensionTabItems
+                            .Where(x => x.Content.GetType().Equals(typeof(DocumentationBrowserView)))
+                            .FirstOrDefault();
+
+            //Validate hat the DocmentationBrowser extension tab is valid
```

Comment:
validate that?

Reply:
@QilongTang 
Fixed in the comment: https://github.com/DynamoDS/Dynamo/pull/11137/commits/d86ba2c296c92207d573903831a776355bf1ba67